addons/hr_recruitment_survey/wizard/survey_invite.py

```diff
@@ -20,6 +19,7 @@ def _send_mail(self, answer):
         mail = super()._send_mail(answer)
         if answer.applicant_id:
             answer.applicant_id.message_post(body=mail.body_html)
+            mail.send(auto_commit=True)
```

Comment:
Not sure about `auto_commit=True` :thinking: I don't think we need it from the description of the parameter :innocent: